agenta-backend/agenta_backend/tests/unit/test_evaluators.py

```diff
@@ -335,56 +442,89 @@ def test_auto_levenshtein_distance(output, data_point, settings_values, expected
 
 
 @pytest.mark.parametrize(
-    "settings_values, expected_min, expected_max",
+    "settings_values, expected_min, openai_api_key, expected_max",
     [
         (
             {
                 "question_key": "rag.retriever.internals.prompt",
                 "answer_key": "rag.reporter.outputs.report",
                 "contexts_key": "rag.retriever.outputs.movies",
             },
+            os.environ.get("OPENAI_API_KEY"),
             0.0,
             1.0,
         ),
+        (
+            {
+                "question_key": "rag.retriever.internals.prompt",
+                "answer_key": "rag.reporter.outputs.report",
+                "contexts_key": "rag.retriever.outputs.movies",
+            },
+            None,
+            None,
+            None,
+        ),
         # add more use cases
     ],
 )
-def test_rag_faithfulness_evaluator(settings_values, expected_min, expected_max):
-    result = rag_faithfulness(
+@pytest.mark.asyncio
+async def test_rag_faithfulness_evaluator(
```

Comment:
@aybruhm -- This one seems to be failing in GitHub Actions. See PR checks.

Reply:
Resolved.